timelock-agent/src/main/java/com/palantir/atlasdb/timelock/paxos/PaxosResourcesFactory.java

```diff
@@ -165,17 +165,20 @@ private static PaxosResources configureLeaderForAllClients(
 
         PingableLeader pingableLeader = factory.components().pingableLeader(PaxosUseCase.PSEUDO_LEADERSHIP_CLIENT);
 
+        LeaderLearnerResource leaderLearnerResource =
+                new LeaderLearnerResource(factory.components().learner(PaxosUseCase.PSEUDO_LEADERSHIP_CLIENT));
+
         return resourcesBuilder
```

Comment:
Quite messy (and registration / derived things for it in PaxosResources.java are as well). Deferring cleanup or how we disable Jersey to after migrating all the things (either we cleanly separate them here, e.g. AddhocResouces -> .jersey, .undertow or stg analogous if possible to NonConjureTimeLockResourcse) or we pass _ -> {} for the jersey registrar in internal wrapper